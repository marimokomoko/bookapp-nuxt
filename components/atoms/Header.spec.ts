import { mount } from '@vue/test-utils'
import Header from '@/components/atoms/Header.vue'
describe('画面レイアウト', ()=>{
    it('スナップショットテスト', ()=>{
        const wrapper = mount(Header)
        expect(wrapper.html()).toMatchSnapshot()
    })
})