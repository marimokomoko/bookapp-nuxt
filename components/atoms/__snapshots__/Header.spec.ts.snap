// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`画面レイアウト スナップショットテスト 1`] = `
"<div>
  <v-navigation-drawer fixed=\\"\\" app=\\"\\">
    <v-list>
      <v-list-item to=\\"/\\" router=\\"\\" exact=\\"\\">
        <v-list-item-action>
          <v-icon>mdi-apps</v-icon>
        </v-list-item-action>
        <v-list-item-content>
          <v-list-item-title>Welcome</v-list-item-title>
        </v-list-item-content>
      </v-list-item>
      <v-list-item to=\\"/book\\" router=\\"\\" exact=\\"\\">
        <v-list-item-action>
          <v-icon>mdi-chart-bubble</v-icon>
        </v-list-item-action>
        <v-list-item-content>
          <v-list-item-title>BookList</v-list-item-title>
        </v-list-item-content>
      </v-list-item>
    </v-list>
  </v-navigation-drawer>
  <v-app-bar fixed=\\"\\" app=\\"\\">
    <v-app-bar-nav-icon></v-app-bar-nav-icon>
    <v-toolbar-title>書籍管理アプリ</v-toolbar-title>
    <v-spacer></v-spacer>
  </v-app-bar>
</div>"
`;
